modules/core/shared/src/test/scala/freestyle/free/free.scala

```diff
@@ -132,13 +188,15 @@ class freeTests extends WordSpec with Matchers {
     }
 
     "using the Applicative instance to combine operations into a FS.Par" in {
-      """
-        import cats.syntax.apply._
-        @free trait X {
-          def a: FS[Int]
-          def b: FS.Par[Int] = (a, a).mapN(_+_)
-        }
-      """ should compile
+      import cats.syntax.apply._
+      @free trait X {
+        def a: FS[Int]
+        def b: FS.Par[Int] = (a, a).mapN(_+_)
+      }
+      object Y extends X.Handler[Id] {
+        def a: Int = 5
+      }
+      //X[X.Op].b.interpret[Id] shouldEqual 10
```

Comment:
Is this comment useful?

Reply:
No, it's a typo. It should be a line to run the derived `b` method with the `Y` Handler.